test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -265,6 +265,9 @@ hw.module @M1<param1: i42>(in %clock : i1, in %cond : i1, in %val : i8) {
       // CHECK-NEXT: $fwrite(32'h80000002, "M: %x\n", `MACRO(8'(val + 8'h2A), val ^ 8'h2A));
       sv.fwrite %fd, "M: %x\n"(%text) : i8
 
+      // CHECK-NEXT: `INIT_RANDOM(val)
+      sv.macro.ref @INIT_RANDOM (%val) : i8
```

Comment:
This test doesn't hit the interleave comma path nor the path without parentheses. Could the PR include a test of these if it is not there already?